bin/server/deploy.sh

```diff
@@ -24,8 +24,8 @@ bin/server/install.sh
 # Rebuild the app.
 bin/build-docker production
 
-# Launch new Rails services.
-docker compose up --detach clock nginx web worker
+# Launch fresh services.
+docker compose up --detach
```

Comment:
Sort of an unrelated change, but I think it's maybe better / more reliable to just relaunch everything. I can imagine that this might possible make resolution of the `depends_on` chains more reliable.